frontend/src/styles/globals.css

```diff
@@ -0,0 +1,20 @@
+:root {
+  --color-blue: #4b00cc;
+  --color-purple: #9f18d9;
+  --color-hot-pink: #ea1889;
+  --color-light-pink: #d396ed;
+  --color-green: #57f2b9;
+  --color-yellow: #fff0ad;
+  --color-black: #000;
+  --color-dark-gray: #404040;
+  --color-gray: #808080;
+  --color-light-gray: #bfbfbf;
+  --color-white: #fff;
+  --font-family-body: 'Work Sans';
```

Comment:
Have you configured your editor to apply Prettier on save? Seeing single quotes here.

Reply:
Also do this font families work like this? Do they have to be imported?

Reply:
Prettier should be applying on save but it's not working since yesterday I believe for some unknown reason